packages/passport/sdk/src/zkEvm/sendDeployTransactionAndPersonalSign.ts

```diff
@@ -0,0 +1,47 @@
+import { prepareAndSignTransaction, pollRelayerTransaction, TransactionParams } from './transactionHelpers';
+import { personalSign } from './personalSign';
+
+type EthSendDeployTransactionParams = TransactionParams & {
+  params: Array<any>;
+};
+
+export const sendDeployTransactionAndPersonalSign = async ({
+  params,
+  ethSigner,
+  rpcProvider,
+  relayerClient,
+  guardianClient,
+  zkEvmAddress,
+  flow,
+}: EthSendDeployTransactionParams): Promise<string> => {
+  const deployTransaction = { to: zkEvmAddress, value: 0 };
+
+  const { relayerId } = await prepareAndSignTransaction({
+    transactionRequest: deployTransaction,
+    ethSigner,
+    rpcProvider,
+    guardianClient,
+    relayerClient,
+    zkEvmAddress,
+    flow,
+  });
+
+  return guardianClient.withConfirmationScreen({
+    width: 480,
+    height: 720,
+  })(async () => {
```

Comment:
These are the default height and width values, so they can be omitted:
```suggestion
  return guardianClient.withConfirmationScreen()(async () => {
```